webrtc.html

```diff
@@ -903,13 +901,87 @@
+                    type of "offer", and it has a version that is later than the
+                    currentRemoteDesc, then the pendingRemoteDesc will be set to
+                    the argument and the state will transition to
+                    have-remote-offer.  </p> </li>
+
+                    <!-- F transition haveRemoteOffer or haveLocalProvAnswer to
+                    stable --> <li><p>If the remote description was set, and the
+                    <code><a>RTCSessionDescription</a></code> argument has a
+                    type of "answer", then this complets an offer answer
+                    negotiation and currentRemoteDesc is set to the argument,
+                    currentLocalDesc is set to the value of pendingLocalDesc,
+                    then pendingRemoteDesc and pendingLocalDesc are set to null,
+                    and the state will transitions to stable.  </p> </li>
+
+                    <!-- B transition rollback to haveLocalOffer to stable -->
+                    <li> <p>If the local description was set, and the 
+                    <code><a>RTCSessionDescription</a></code> argument has a 
+                    type of "rollback", then this is a rollback and the 
+                    pendingLocalDesc is set to null and the state will 
+                    transition to stable.  </p> </li>
+
+                    <!-- E transition rollback to haveRemoteOffer to stable -->
+                    <li> <p>If the remote description was set, and the 
+                    <code><a>RTCSessionDescription</a></code> argument has a 
+                    type of "rollback", then this is a rollback and the 
+                    pendingRemoteDesc is set to null and the state will 
+                    transition to stable.  </p> </li>
+
+                    <!-- G transition haveRemoteOffer to haveLocalProvAnswer -->
+                    <li><p>If the local description was set, and the
+                    <code><a>RTCSessionDescription</a></code> argument has a
+                    type of "pranswer", then pendingLocalDesc will be set to the
+                    argument and the state will transitions to
+                    have-local-pranswer unless the state was stable,
+                    have-local-offer, or have-remote-pranswer in which case an
+                    InvalidStateError error will be generated.  </p> </li>
+
+                     <!-- H transition haveRemoteOffer to haveLocalProvAnswer
+                    --> <li><p>If the remote description was set, and the
+                    <code><a>RTCSessionDescription</a></code> argument has a
+                    type of "pranswer", then pendingRemoteDesc will be set to
+                    the argument and the state will transitions to
+                    have-remote-pranswer unless the state was stable,
+                    have-remote-offer, or have-local-pranswer in which case an
+                    InvalidStateError error will be generated.  </p> </li>
```

Comment:
I think I would rather rephrase the earlier error check as having this here helps keep the flow in the right place but I'm sort of missing where the earlier one happens. Perhaps it can be clarified a bit. Don't really care deeply about where the text is as long as it is clear this causes an error.